cpp/ps0003/problem 3.cpp

```diff
@@ -1,12 +1,16 @@
 #include <iostream>
 using namespace std;
 int main(){
+    // prompting user for number 2/2
     int number;
     cout << "enter number \n";
     cin >> number;
-    if (number %2 ==0)
-    cout << "number is even \n";
+
+    // checking if the number is odd or even 3/3
+    if (number % 2 == 0)
+        cout << "number is even \n";
     else
-    cout << "number is odd \n";
+        cout << "number is odd \n";
```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console